README.md

```diff
@@ -100,9 +100,14 @@ check an entire directory. Let's clone and check the directory of one of the
 maintainers:
 
 ```bash
-git clone https://github.com/SuperKogito/SuperKogito.github.io.git
-cd SuperKogito.github.io
-urlchecker check .
+$ git clone https://github.com/SuperKogito/SuperKogito.github.io.git
+$ cd SuperKogito.github.io
+```
+
+and run the simplest command to check the present working directory (.).
+
+```bash
+$ urlchecker check .
 
 $ urlchecker check .
```

Comment:
I am not sure if this is intentional but `$ urlchecker check  .` is written twice.

Reply:
Second one is removed, that was definitely a mistake!